tetris.py

```diff
@@ -125,29 +132,63 @@ def new_block(self):
             self.current_block_shape = self.upcoming_block_shape
         self.current_block_center = (int(game_width / 2), -2)
         self.upcoming_block_shape = random.choice(shape_names)
+        self.rotate_counter = 0
 
     def get_current_block(self):
         (x, y) = self.current_block_center
         if self.current_block_shape == "I":
-            return [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)]
+            I = [
+                [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)],
+                [(x, y + 1), (x, y), (x, y - 1), (x, y - 2)],
+            ]
+            return I[self.rotate_counter % len(I)]
         if self.current_block_shape == "L":
-            return [(x - 1, y - 1), (x - 1, y), (x, y), (x + 1, y)]
+            L = [
+                [(x - 1, y + 1), (x, y + 1), (x + 1, y + 1), (x + 1, y)],
+                [(x - 1, y - 1), (x - 1, y), (x - 1, y + 1), (x, y + 1)],
+                [(x - 1, y - 1), (x, y - 1), (x + 1, y - 1), (x - 1, y)],
+                [(x, y - 1), (x + 1, y - 1), (x + 1, y), (x + 1, y + 1)],
```

Comment:
Same here, changing so that below each coordinate is a rotated version of that coordinate:

```suggestion
                [(x + 1, y - 1), (x, y - 1), (x - 1, y - 1), (x - 1, y)],
                [(x + 1, y + 1), (x + 1, y), (x + 1, y - 1), (x, y - 1)],
```